packages/components/dialog/src/dialog.ts

```diff
@@ -47,17 +67,38 @@ export class Dialog extends ScopedElementsMixin(LitElement) {
         @cancel=${this.#onCancel}
         @click=${this.#onClick}
         @close=${this.#onClose}
+        @keydown=${this.#onKeydown}
         .role=${this.role}
+        aria-labelledby="title"
         part="dialog"
       >
         <slot name="header">
-          <slot name="title"></slot>
+          <slot name="titles">
```

Comment:
`aria-labelledby` will no longer work if you slot something in the `titles` slot. Not sure if/how we should handle that....

Reply:
Maybe we don't nedd `titles` slot, maybe slot `title` and `subtitle` would be enough.

This is how it looks like in Figma:

<img width="1046" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/cd9a5431-6983-4fc3-881a-ecfb536d5d24">

Reply:
I agree: I think a `header` slot is enough.